detox/android/detox/src/main/java/com/wix/detox/config/DetoxConfig.kt

```diff
@@ -0,0 +1,14 @@
+package com.wix.detox.config
+
+class DetoxConfig {
+    var idlePolicyConfig: DetoxIdlePolicyConfig? = DetoxIdlePolicyConfig()
```

Comment:
if the ctor does not throw exceptions, then no need for Nullable type, you can remove `?` if calling constructor of `DetoxIdlePolicyConfig` does not yield null.

Reply:
and can be val if the reference is not changing

Reply:
as i see the usage you are creating an object and use the setter 
```
class DetoxConfid(val idlePolicyConfig: DetoxIdlePolicyConfig = DetoxIdlePolicyConfig(),var rnContextLoadTimeoutSec:Int = 60){

fun apply() {...}
companio object {.....}
```

Reply:
oops, right about the null heh

Reply:
Ok I kept fields as `var` and applied `@JvmField` on all fields... this is what I really wanted to begin with... thanks for pointing this out!!!